bool get_data
---
simulator_gazebo/CamModuleInfo[] module_info
